egs/wsj/s5/utils/validate_data_dir.sh

```diff
@@ -125,12 +125,6 @@ fi
 
 num_utts=`cat $tmpdir/utts | wc -l`
 if ! $no_text; then
```

Comment:
can remove the --no-print option, which was introduced with the PR that added this.

Reply:
removed